pkg/tid/cli/command/timesheet/list.go

```diff
@@ -47,7 +49,12 @@ func ListCommand(factory util.Factory) *console.Command {
 		now := timeutil.Date(time.Now())
 
 		if !hasStart {
-			start = timeutil.LastWeekday(time.Monday)
+			weekday, err := timeutil.StringToWeekday(config.Display.FirstWeekDay)
+			if err != nil {
+				fmt.Println(err)
```

Comment:
If it's in a command's execute function then it can just do `return err`, and the console lib will handle outputting the error and setting an exit code automatically.

In some cases there are "soft errors" so to speak where you just want to show a message but not really say "this was an error" to the shell, more like a warning really. In those cases I just `output` to print the message and return nil straight after to stop processing the command.